scripts/check-links.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# Check HTTP status codes of links in the given files.
+# Success: 2xx, Errors: 4xx/5xx, Warnings: anything else.
+# Redirects (3xx) are followed.
+# Usage:
+#     check_links.sh [<files>]
+
+base_url="${CHECK_LINKS_RELATIVE_URL:-https://dvc.org}"
```

Comment:
consider adding `set -euxo pipefail` into all scripts, it's a good practice to have it